script.js

```diff
@@ -48,11 +48,16 @@ function hideLoader() {
   loader.style.display = 'none';
 }
 
+getOpenEuroUrl = () => {
```

Comment:
I'm getting console error that reads in firefox `ReferenceError: assignment to undeclared variable getOpenEuroUrl` which makes sense to be as this line should probably be `const getOpenEuroUrl = () => {` 

Though I'd prefer if you stuck to the convention in the rest of the file and use `function getOpenEuroUrl`.

I also don't understand the name of this function, is it supposed to be `getOpenNeuroUrl`?